sunpy/coordinates/utils.py

```diff
@@ -384,3 +384,45 @@ def get_rectangle_coordinates(bottom_left, *, top_right=None,
             top_right = top_right.frame
 
     return bottom_left, top_right
+
+
+def solar_angle_equivalencies(observer):
```

Comment:
```suggestion
def solar_angle_equivalency(observer):
```
There's only one equivalency here...